integration/client/container_fuzzer.go

```diff
@@ -395,8 +395,8 @@ func doFuzz(data []byte, shouldTearDown bool) int {
 	return 1
 }
 
-// FuzzCreateContainerNoTearDown() implements a fuzzer
-// similar to FuzzCreateContainerWithTearDown() and
+// FuzzNoTearDownWithDownload() implements a fuzzer
+// similar to FuzzCreateContainerNoTearDown() and
 // FuzzCreateContainerWithTearDown(), but it takes a
```

Comment:
It looks like I've missed some history here, since this isn't the only one where it isn't matching, but the function name is FuzzIntegNoTearDownWithDownload, not FuzzNoTearDownWithDownload.

For this to fit [conventional Go style](https://go.dev/doc/comment#func), it'd also need to change like this:

```suggestion
// FuzzIntegNoTearDownWithDownload implements a fuzzer
// similar to FuzzIntegCreateContainerNoTearDown and
// FuzzIntegCreateContainerWithTearDown, but it takes a
```

Reply:
Thanks for your review. I have rechecked the whole file following your suggestion, and updated the rest of the function names :)